<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>record</class>
 <widget class="QDialog" name="record">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="headLabel">
   <property name="geometry">
    <rect>
     <x>325</x>
     <y>10</y>
     <width>170</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Video preview</string>
   </property>
  </widget>
  <widget class="QListView" name="listView">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>90</y>
     <width>161</width>
     <height>461</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="showVideo">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>70</y>
     <width>640</width>
     <height>480</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="playButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>580</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Play</string>
   </property>
  </widget>
  <widget class="QPushButton" name="quitButton">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>580</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QLabel" name="videoName">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>550</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
